iib/web/static/api_v1.yaml

```diff
@@ -805,6 +810,11 @@ components:
             non-privileged users to use overwrite_from_index. The format of the token must be in
             the format "user:password".
           example: token
+        registry_auths:
+          type: string
```

Comment:
@lcarva I used string type for this parameter. On web it's displayed like this: 
```
{
"add_arches": [
"amd64",
"s390x"
],
"binary_image": "quay.io/operator-framework/upstream-registry-builder:v1.5.9",
"bundles": [
"quay.io/user/openshift-bundle-example-container:v1"
],
"cnr_token": "token",
"deprecation_list": [
"quay.io/user/openshift-bundle-example-container@sha256:29e4531e4625653"
],
"force_backport": false,
"from_index": "quay.io/iib-stage/iib:4",
"organization": "openshift-operators",
"overwrite_from_index": false,
"overwrite_from_index_token": "token",
"registry_auths": "{\"auths\": {\"registry.example.io\": {\"auth\": \"my-secret-token-base-64-encoded\"}}}",
"distribution_scope": "prod"
}
```
If correct registry and secret is provided in this format ^ . It works well. Let me know if yo uhave better idea how to deal with passing json.

Reply:
I think it's odd to inline a json inside another json. Can we use a json object instead of string?